micro-ui/web/micro-ui-internals/packages/react-components/src/molecules/DashboardBox.js

```
@@ -1,10 +1,11 @@
 import React from "react";
 import { Link } from "react-router-dom";
+import { COLOR_FILL } from "../atoms/contants";
 
 const ArrowRight = ({ to }) => (
   <Link to={to}>
     <svg style={{ display: "inline", height: "24px" }} xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 32 32" fill="none">
```

Biome

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)